packages/core/src/module.ts

```diff
@@ -43,7 +43,7 @@ export function createModule(options: CreateModuleOptions): RuntimeModule {
       description: options.metadata.description || '',
     },
     spec: {
-      properties: {},
+      properties: { type: 'object' },
```

Comment:
Why need this?

Reply:
Because the `json-schema-editor` provides two schema formats, array and object         
but our app only seems to use `object`, we need a default value like `Type.Object({})`, or `{type:'object'}`

Reply:
I think the array should be removed from thejson-schema-editor afterwards. now it is judged on sunmao's side

Reply:
Better add a comment here to explain this.